network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/video-thumb.html

```diff
@@ -0,0 +1,7 @@
+{% load i18n static %}
+<div class="tw-relative video-thumb {% if size %} video-thumb--{{ size }} {% endif %} {{ class }}" data-scroll data-scroll-speed="{{ speed }}">
+  <div class="tw-flex tw-flex-col video-thumb__inner">
+    <img class="timeline__scene-image" src="{% static '_images/youtube-regrets/regrets-reporter/2022-timeline/' %}{{ image }}" alt="">
+    <p class="tw-text-[0.75rem] tw-text-sm tw-font-bold tw-mt-2">{% trans text %}</p>
```

Comment:
The `{% trans %}` tag doesn’t accept content passed dynamically, we need to localize it first in timeline.html as variables, *then* pass them through.
That being said, no need to fix this now, I’ll handle it in a future localization PR

Reply:
Thanks @TheoChevalier - I'll resolve this for now in that case.